src/Nethermind/Nethermind.JsonRpc/Modules/DebugModule/DebugRpcModule.cs

```diff
@@ -272,5 +273,10 @@ public ResultWrapper<bool> debug_resetHead(Keccak blockHash)
             _debugBridge.UpdateHeadBlock(blockHash);
             return ResultWrapper<bool>.Success(true);
         }
+
+        public Task<ResultWrapper<SyncReportSymmary>> debug_getSynsStage()
```

Comment:
Typo "syns" but should be "syncs"

Reply:
Either `syncStage` or `syncsStages`?

Reply:
Maybe `admin` would be better than `debug` ?

Reply:
@LukaszRozmej @DemuirGos 
I'd suggest putting that somewhere under Health or something similar which is enabled by default on nodes.

Thios endpoint can be used not only by us but also by users or on support to ensure without logs at what stage of sync we are right now.

WDYT?

Reply:
Moving to module enabled by default would also help me a bit with tests since now I need to ensure I have it enabled on each node :D